crm crash during cert renewal

CRM crashed during LB cert renewal (domain names have been redacted):
```
2024-05-30T07:51:13.769Z        INFO    6c13b42c2f461e03        certs/certifications.go:291     write proxy certs to rootLB     {"lbname": "q-b1-q-dev.b-m-t.e.app.xyz.edgexr.org"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x1cd94ae]

goroutine 145 [running]:
github.com/edgexr/edge-cloud-platform/pkg/proxy/certs.(*ProxyCerts).writeCertToRootLb(0xc00049b380, {0x2aa86f0, 0xc00067fa10}, 0xc0018c3970, {0x0, 0x0}, {0xc001b67900, 0x41})
        github.com/edgexr/edge-cloud-platform/pkg/proxy/certs/certifications.go:292 +0xee
github.com/edgexr/edge-cloud-platform/pkg/proxy/certs.(*ProxyCerts).refreshCerts(0xc00049b380, {0x2aa86f0, 0xc00067fa10})
        github.com/edgexr/edge-cloud-platform/pkg/proxy/certs/certifications.go:255 +0xde7
github.com/edgexr/edge-cloud-platform/pkg/proxy/certs.(*ProxyCerts).Start.func1()
        github.com/edgexr/edge-cloud-platform/pkg/proxy/certs/certifications.go:138 +0x12c
created by github.com/edgexr/edge-cloud-platform/pkg/proxy/certs.(*ProxyCerts).Start in goroutine 90
        github.com/edgexr/edge-cloud-platform/pkg/proxy/certs/certifications.go:135 +0x79
2024-05-30T07:51:16.118Z        INFO    notrace node/nodemgr.go:116     start main nodeMgr init
```

CRM version: ghcr.io/edgexr/edge-cloud-crm:2024-03-28.1

--- a/pkg/proxy/certs/certifications.go
+++ b/pkg/proxy/certs/certifications.go
@@ -254,6 +254,11 @@ func (s *ProxyCerts) refreshCerts(ctx context.Context) error {
 		// apply new cert
 		for _, lbname := range lbNames {
 			lbClient := lbClients[lbname]
+			if lbClient.Client == nil {
+				log.SpanLog(ctx, log.DebugLevelInfra, "refreshCerts skipping unexpected nil client", "lbname", lbname, "fqdn", lbClient.FQDN)
+				errs = append(errs, "no ssh client for "+lbname)
+				continue
+			}
 			err = s.writeCertToRootLb(ctx, &cert, lbClient.Client, lbname)
 			if err != nil {
 				log.SpanLog(ctx, log.DebugLevelInfra, "failed to write cert to lb", "wildcardName", wildcardName, "lbname", lbname, "err", err)

